lib/src/bindings/js_extension.dart

```diff
@@ -0,0 +1,438 @@
+// ignore_for_file: public_member_api_docs
+
+import 'dart:js_interop';
```

Comment:
Here I don't know well if I choosen the right solution to interop with JS. All the functions compiled to WASM are exposed globally. I think there is a way to put them inside an `extension type` but I didn't find a solution.